### ISSUE TITLE
[✨] Qwikify for NextJS through proxy

### ISSUE THREAD

**Issue by mhevery** [BuilderIO/qwik#2304]
### Is your feature request related to a problem?

I would like to try using Qwik in my existing NextJS project in an incremental way.

### Describe the solution you'd like

1. Create NextJS project.
2. Create a Qwik project inside the NextJS project.
3. Choose an existing NextJS route and include a Qwik route inside that location by proxying the response.
4. Ability to use React components inside the Qwik routes through React Qwikfy

See this [loom](https://www.loom.com/share/df902dcebdb241a196dbf827b6e85436) for more background:

File: `pages/test-page.tsx`
```typescript
// Helper library which allows fetching HTML from Qwik URL
import { getQwikRoute } from "@builder.io/qwik-react/proxy";

export async function getStaticProps(context) {
  return {
    props: {
      qwikHtml: await getQwikRoute("/test-page"), // Fetch HTML from Qwik
    },
  };
}

// Render NextJS page with Qwik HTML
export default function Home({ qwikHtml }: { qwikHtml: string }) {
  return <div dangerouslySetInnerHTML={{ __html: qwikHtml }} />;
}
```

File: `qwik-app/src/routes/test-page/index.tsx`
```typescript
import { component$ } from "@builder.io/qwik";
import { ReactComp } from "./qwikify";

export default component$(() => {
  return (
    <>
      <span>Hello Qwik</span>
      <ReactComp />
    </>
  );
});

```

### Describe alternatives you've considered

n/a

**Comment by gioboa** [comment]
Hi @thejackshelton, are you working on this? :thinking:

**Comment by thejackshelton-kunaico** [comment]
Yeah so I got it to resume, but then I think Next.js didn't understand Qwik's signals and things broke

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
